src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -171,6 +173,33 @@ class DashboardWidgetQuerySerializer(CamelSnakeSerializer[Dashboard]):
 
     validate_id = validate_id
 
+    def get_metrics_features(
```

Comment:
Should I move this up to the endpoint class and pass `has_metrics` along as context from up there?